code/src/beetle.cpp

```diff
@@ -0,0 +1,75 @@
+#include <beetle.hpp>
+
+namespace r2d2::moving_platform {
+
+    beetle_c::beetle_c(r2d2::usart::usart_ports &usart_port, unsigned int baud_rate,
```

Comment:
Same as before why is it grouped like this and not all with a line end after the comma?

Reply:
Your commit has fixed this